app/scss/style.scss

```diff
@@ -85,6 +110,17 @@
       font-size: calc(14px + 1vw);
     }
   }
+  .specialties-list {
+    &:after {
+      content: "";
+      flex: auto;
+    }
+    li {
+      width: calc(33.33333% - 8px);
+      font-size: 0.9em;
+      text-align: center;
+    }
+  }
 }
 
 @media screen and (max-width: 750px) {
```

Comment:
ditto

Reply:
Though I realize this one wasn't originally done that way.

Reply:
Done